tools/mock-service-host/README.md

```diff
@@ -113,16 +107,126 @@ If failed for permission reasons, please contact [CodeGen Core team vsccodegen@m
 Common trouble shootings for starting the web server:
 
 -   Make sure all ports used in Mock Service Host haven't been used by other processes.
--   Try to used sudo/"start as administrator" if failed to start listening.
+-   Try to use sudo/"run as administrator" if failed to start listening.
+
+It takes up to two minutes to load all swagger files in the azure-rest-api-specs repo after the mock server started. When loading finished, a log with "validator initialized" is shown in the console.
```

Comment:
can it be consistent to use mock service host?

Reply:
Do you mean the use `root` part?

Reply:
I mean the term of 'mock service host'.
Seems in some places, we use mock service host and in other places we use mock server host